data/scripts/field_move_scripts.inc

```diff
@@ -233,16 +220,12 @@ EventScript_UseDive:: @ 8290B0F
 	bufferpartymonnick 0, VAR_RESULT
 	setfieldeffectargument 0, VAR_RESULT
 	setfieldeffectargument 1, 1
-	msgbox Text_WantToDive, MSGBOX_YESNO
-	compare VAR_RESULT, NO
-	goto_if_eq EventScript_EndDive
-	msgbox Text_MonUsedDive, MSGBOX_DEFAULT
```

Comment:
These deletions simply speed up HM scripts. Now you just press A while sat on a dive tile and boom you're underwater. Similar changes throughout this file.